test/fuzzing/run-draw-fuzzer-tests.py

```diff
@@ -1,66 +1,53 @@
 #!/usr/bin/env python3
 
-import sys, os, subprocess, tempfile, shutil
+import sys
+import os
+from hb_fuzzer_tools import (
+    run_command,
+    chunkify,
+    find_fuzzer_binary,
+    gather_files
+)
 
+def main():
+    srcdir = os.getenv("srcdir", ".")
+    EXEEXT = os.getenv("EXEEXT", "")
+    top_builddir = os.getenv("top_builddir", ".")
 
-def cmd (command):
-	# https://stackoverflow.com/a/4408409 as we might have huge output sometimes
-	with tempfile.TemporaryFile () as tempf:
-		p = subprocess.Popen (command, stderr=tempf)
+    # Find the fuzzer binary
+    default_bin = os.path.join(top_builddir, "hb-draw-fuzzer" + EXEEXT)
+    hb_draw_fuzzer = find_fuzzer_binary(default_bin, sys.argv)
```

Comment:
May be it is time to simplify this and drop `default_bin` entirely? We always pass the fuzzer binary to meson and I don’t anyone runs these scripts manually.

Reply:
Agreed. I don't know meson. Help :D.